components/normalizer/src/lib.rs

```diff
@@ -1808,6 +1808,9 @@ impl<'data> DecomposingNormalizerBorrowed<'data> {
                             // Fast-track succeeded!
                             continue 'fastest;
                         }
+                        // This deliberately isn't panic-free, since the code pattern
+                        // that was OK for the composing counterpart regressed
+                        // English and French performance if done here, too.
```

Comment:
question: what's the code pattern being talked of here?

Can `.get(...).unwrap_or_default()` or something similar work? We really should avoid panicky code.

Reply:
Not, AFAICT, in a way that I could reason about not to have worse outcomes than panic if the code ever runs.